docs/marketplace/dev/glue-api-guides/202200.0/shopping-lists/managing-shopping-lists.md

```diff
@@ -0,0 +1,1127 @@
+                            "id": "136_24425591"
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "type": "concrete-products",
+            "id": "005_30663301",
+            "attributes": {
+                "sku": "005_30663301",
+                "isDiscontinued": false,
+                "discontinuedNote": null,
+                "averageRating": null,
+                "reviewCount": 0,
+                "name": "Canon IXUS 175",
+                "description": "Creative play Play with your creativity using a range of Creative Filters. Re-create the distortion of a fish-eye lens, make scenes in stills or movies look like miniature scale models and much more. Capture the stunning detail in everyday subjects using 1 cm Macro to get right up close. Enjoy exceptional quality, detailed images thanks to 20.0 Megapixels and DIGIC 4+ processing. Face Detection technology makes capturing great shots of friends effortless, while Auto Zoom intelligently helps you select the best framing at the touch of a button.",
+                "attributes": {
+                    "optical_zoom": "8 x",
+                    "combined_zoom": "32 x",
+                    "display": "LCD",
+                    "hdmi": "no",
+                    "brand": "Canon",
+                    "color": "Blue"
+                },
+                "superAttributesDefinition": [
+                    "color"
+                ],
+                "metaTitle": "Canon IXUS 175",
+                "metaKeywords": "Canon,Entertainment Electronics",
+                "metaDESCRIPTION": "Creative play Play with your creativity using a range of Creative Filters. Re-create the distortion of a fish-eye lens, make scenes in stills or movies loo",
+                "attributeNames": {
+                    "optical_zoom": "Optical zoom",
+                    "combined_zoom": "Combined zoom",
+                    "display": "Display",
+                    "hdmi": "HDMI",
+                    "brand": "Brand",
+                    "color": "Color"
+                }
+            },
+            "links": {
+                "self": "http://glue.mysprykershop.com/concrete-products/005_30663301"
+            }
+        },
+        {
+            "type": "shopping-list-items",
+            "id": "00fed212-3dc9-569f-885f-3ddca41dea08",
+            "attributes": {...},
+            "links": {....},
+            "relationships": {
+                "concrete-products": {
+                    "data": [
+                        {
+                            "type": "concrete-products",
+                            "id": "005_30663301"
+                        }
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+</details>  
+
+<details>
+<summary markdown='span'>Response sample: retrieve a shopping list with its items (marketplace products and product offers)</summary>
```

Comment:
The response sample is not complete, it does not contain product offers & marketplace products.